server/nix/default.nix

```diff
@@ -60,7 +60,14 @@ pkgs.haskell-nix.cabalProject {
 
   extraSources = [
     {
-      src = inputs.ogmios.inputs.cardano-base;
+      src = inputs.cardano-addresses;
```

Comment:
All of these changes need to be reverted. I switched to using Ogmios' inputs since it's a lot more convenient